tray/menu.go

```diff
@@ -89,10 +89,7 @@ func addVpnSection(ti *Instance) {
 	mStatus.Disable()
 
 	if ti.state.vpnStatus == ConnectedString {
-		vpnServerName := ti.state.vpnName
-		if vpnServerName == "" {
```

Comment:
Not needed anymore?

Reply:
I've created a function `serverName()` that calculates the server name, because there are too many `if` to put in each place :smile: 
This code must be into that function.